popup.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="popup.css">
+   
+  </head>
+  <body>
+    <div class="container">
+      <form id="form">
+        <div class="textarea_style">
+          <textarea class="textarea" id="textarea" name="textarea_description" rows="4" cols="37" maxlength="100000" minlength="3" placeholder="Write your note here..."></textarea>
+        </div>
+
+      </form>
+     
+    </div>
+    <button class="tooltip" id="add"><img src="https://img.icons8.com/material-outlined/15/000000/plus--v1.png"/>
+      <span class="tooltiptext">add note</span>
+    </button>
+   
+    <button class="tooltip" id="search"><img src="https://img.icons8.com/material-outlined/15/000000/search--v1.png"/></a>
+      <span class="tooltiptext">search web</span>
+    </button>
+        <button class="tooltip" id="download"><img src="https://img.icons8.com/material-rounded/15/000000/download--v1.png"/></a>
+      <span class="tooltiptext">download all</span>
+    </button>
+    <button class="tooltip" id="clear"><img src="https://img.icons8.com/material-rounded/15/000000/delete-forever.png"/></a>
+      <span class="tooltiptext">delete all</span>
+    </button>
+    <button class="tooltip" id="highlight"><img src="https://img.icons8.com/ios-glyphs/15/000000/marker-pen.png"/></a>
```

Comment:
Perhaps you could include these icons locally in your project? Referencing images on other websites is a great way to ensure your application suddenly breaks one day when those images get moved or the site goes down 😀